### PR TITLE
Added sponsors!

### PR DESCRIPTION
**Pull request by SPLATPLAYS**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -86,7 +86,10 @@ <h2 class="text-3xl font-bold mb-4">aliquet praesent sodades</h2>
     <section class="blog bg-gray-200 py-16">
         <div class="container mx-auto">
             <h2 class="text-3xl font-bold mb-4">Our Sponsors!</h2>
-            <!-- Sponsor info goes here -->
+<!--              <p class="text-lg"><a href="https://www.wa.gov.au/organisation/department-of-primary-industries-and-regional-development">Department of primary industries and regional development</a></p> -->
```
Comment (spacey-sooty) [inline index.html:89]: why

File: index.html [hunk 2]
```diff
@@ -86,7 +86,10 @@ <h2 class="text-3xl font-bold mb-4">aliquet praesent sodades</h2>
...
+            <a href="https://www.wa.gov.au/organisation/department-of-primary-industries-and-regional-development">
```
Comment (spacey-sooty) [inline index.html:90]: should be dynamic for mobile and whatnot

**Review by spacey-sooty — CHANGES_REQUESTED** [review]

File: index.html (hunk 2 above)
Comment (SPLATPLAYS) [reply index.html:90]: Will maybe probably kind of fix later for mobile issue

File: index.html (hunk 1 above)
Comment (SPLATPLAYS) [reply index.html:89]: It doesn't lower performance whatsoever, don't worry about it

File: index.html (hunk 2 above)
Comment (spacey-sooty) [reply index.html:90]: Works for me

File: index.html (hunk 1 above)
Comment (SPLATPLAYS) [reply index.html:89]: Its dashed out text for might be future sponsors

Comment (spacey-sooty) [reply index.html:89]: Its a comment that adds no value, remove it

**Comment by SPLATPLAYS** [comment]
fixed that thing

**Review by spacey-sooty — APPROVED** [review]

**Pull request merged**